packages/langium-vscode/src/language-server/grammar-workspace-manager.ts

```diff
@@ -36,18 +33,18 @@ export class LangiumGrammarWorkspaceManager extends DefaultWorkspaceManager {
     }
 
     override async initializeWorkspace(folders: WorkspaceFolder[], cancelToken = CancellationToken.None): Promise<void> {
-        const buildConf: WorkspaceManagerConf = await this.configurationProvider.getConfiguration(LangiumGrammarLanguageMetaData.languageId, CONFIG_KEY);
+        const buildConf: WorkspaceManagerConf = await this.configurationProvider.getConfiguration('langium', CONFIG_KEY);
         const ignorePatterns = buildConf.ignorePatterns?.split(',')?.map(pattern => pattern.trim())?.filter(pattern => pattern.length > 0);
-        this.matcher = ignorePatterns ? ignore().add(ignorePatterns) : undefined;
+        this.matcher = ignorePatterns ? ignore.default().add(ignorePatterns) : undefined;
```

Comment:
The `ignore` package has a weird export structure, that isn't correctly picked up for ESM code (since the `exports` fields are missing) see source code:

```js
const factory = options => new Ignore(options)

// Fixes typescript
factory.default = factory

module.exports = factory
```